{"name": "test_login_with_invalid_email_and_valid_password_Duplicate", "status": "passed", "start": 1695709774979, "stop": 1695709777952, "uuid": "6845fa00-58da-4e27-a4a5-001bdf53fb96", "historyId": "97eea31a395f9e802bec8794824b1cfb", "testCaseId": "97eea31a395f9e802bec8794824b1cfb", "fullName": "Tests.test_Login(duplicate).TestPanarayLoginDuplicate#test_login_with_invalid_email_and_valid_password_Duplicate", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login(duplicate)"}, {"name": "subSuite", "value": "TestPanarayLoginDuplicate"}, {"name": "host", "value": "WONIT11L32CV6Z"}, {"name": "thread", "value": "12312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login(duplicate)"}]}